xagon-react-app/src/components/GameComponent/inputManager/InputManager.tsx

```diff
@@ -1,23 +1,20 @@
 import {
-  Vector3,
   Scene,
   PointerEventTypes,
-  TransformNode,
   ArcRotateCamera,
   AbstractMesh,
 } from '@babylonjs/core';
 
 import Triangle from 'models/Triangle';
-
-import { getAssetMesh } from 'utils/scene';
+import FlipGesture from './FlipGesture';
 import setupInput from './setupInput';
 
-const TRIANGLE_SCALE = 0.85;
-
 class InputManager {
   //
   private scene: Scene;
 
+  private gesturesMap: Record<number, FlipGesture> = {};
```

Comment:
il tipo qui dovrebbe esser Gesture, non FlipGesture

Reply:
ti consiglio inoltre per chiarezza di definire un tipo GestureId o GestureKey che è un alias di number in questo caso da usare al suo posto